### ISSUE TITLE
Alarm skill doesn't speak the date naturally

### ISSUE THREAD

**Issue by Wolfgange3311999** [MycroftAI/mycroft-core#158]
What I mean by this is if setting an alarm for 5 minutes, it reads the entire date (ie. "set an alarm for Thrusday, June 16th, at 04:05 pm") rather than simply saying the same thing in simpler terms (ie. "set an alarm for 5 minues"). And if the date is a day ahead I think it would be best that it says "Tomorrow at 4:05pm".

**Comment by ethanaward** [comment]
This should be a relatively simple change in the alarm skill.

**Comment by ethanaward** [comment]
This is actually behavior from the base `ScheduledCRUDSkill` class that @jdorleans created, which formats the time metadata as the full date. In order to change that, we'd have to change the base class, which would modify the behavior of all skills that inherit from that. That would be fine as long as we make sure that's what we want to do.
